test/key1.prv

```diff
@@ -0,0 +1 @@
+d9c5dc47ed678fc3e63249953866d79e5cf48418e79d8eec1a985be7393ef3b9
```

Comment:
Only cosmetic, but can we rename the files `test/key0.prv`, `test/key1.prv` and `test/password` to something looking less like exposed secrets? Might undermine trust in the project from someone superficially scanning the codebase. 

Maybe: 

`test/key0.prv` => `test/dev-example-key0.prv`
`test/key1.prv` => `test/dev-example-key1.prv`  
`test/password` =>  `test/dev-example-password`

Reply:
https://github.com/Snowfork/snowbridge/pull/492